### DISCUSSION TITLE
NPM Updates?

### DISCUSSION THREAD

**Discussion by cnatale** [rjsf-team/react-jsonschema-form#3346] [General]
I notice the project's NPM registry hasn't been updated in roughly three years. Are the non-beta releases published to the repository considered deployment ready? Any chance of starting NPM releases again? I ask because the company I work for relies on the NPM registry for keeping packages up to date. Thanks!

**Comment by nickgros** [comment]
@cnatale We publish to npm under the `@rjsf` team, so e.g. `@rjsf/core`, `@rjsf/material-ui`, etc. 

@epicfaace are we able to put a "deprecated, use @rjsf/core" message on the `react-jsonschema-form` npm page?
